recipes/yaxt/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "yaxt" %}
+{% set version = "0.11.2" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.dkrz.de/dkrz-sw/{{ name }}/-/archive/v{{ version }}/{{ name }}-v{{ version }}.tar.gz
+  sha256: e9a552641c3b52daed4d09a59ac879cbe7253acf8cb6d6855211f68c4d4f2227
+
+build:
+  skip: true  # [win]
+  number: {{ build }}
+
+  # add build string so packages can depend on mpi variants explicitly:
+  # `yaxt * mpi_mpich_*` for mpich
+  # `yaxt * mpi_openmpi_*` for openmpi
+  # `yaxt * mpi_*` for any mpi
+
+  {% set mpi = mpi or "mpich" %}
+  {% set mpi_prefix = "mpi_" + mpi %}
+  string: {{ mpi_prefix }}_h{{ PKG_HASH }}_{{ build }}
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
+    - {{ name }} * {{ mpi_prefix }}_*
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+    - {{ stdlib('c') }}
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+    - autoconf
+    - automake
+    - libtool
+    - make
+    - pkg-config
+  host:
+    - {{ mpi }}
+    - "{{ mpi }}-mpifort"
+  run:
+    - {{ mpi }}
+    - "{{ mpi }}-mpifort"
```

Comment:
The *-mpifort packages are for user convenience not for use in recipes. It is highly unlikely that you should need to include the fortrain compiler as a host and run dependency.



https://conda-forge.org/docs/maintainer/knowledge_base/#mpi-compiler-packages